stable_baselines/acer/acer_simple.py

```diff
@@ -670,6 +671,11 @@ def run(self):
         mb_mus = np.asarray(mb_mus, dtype=np.float32).swapaxes(1, 0)
         mb_dones = np.asarray(mb_dones, dtype=np.bool).swapaxes(1, 0)
 
+        if len(mb_action_masks) != 0:
+            if isinstance(self.env.action_space, Discrete):
+                action_mask = np.array([1] * self.env.action_space.n, dtype=np.bool)
+                for i in range(0, self.n_env):
+                    mb_action_masks.insert(0, action_mask)
```

Comment:
If the environment gives us action masks, prepend the first self.n_env ones to the front (all of which are all 1s)